lib/generate.js

```diff
@@ -51,6 +52,24 @@ async function generateMarkup () {
     });
 }//generateMarkup()
 
+async function generateJSON () {
```

Comment:
This function will take the `*.json5` files and convert them all to plain JSON in the generated output. This will be used to populate data values in VueJS.